src/pages/community-champions.js

```diff
@@ -18,6 +26,13 @@ const APPLY_BUTTON_MOBILE_BOTTOM_MARGIN = '12px';
 const WHAT_IT_MEANS_TO_BE_A_CHAMPION_ROW_GAP = '96px';
 const WHAT_IT_MEANS_TO_BE_A_CHAMPION_MOBILE_ROW_GAP = '40px';
 const WHAT_IT_MEANS_TO_BE_A_CHAMPION_TEXT_HORIZONTAL_MARGIN = '20%';
+const LEAD_BY_EXAMPLE_CONTAINER_PADDING = '48px';
+const LEAD_BY_EXAMPLE_ICON_WIDTH = '44px';
+const LEAD_BY_EXAMPLE_ICON_MOBILE_WIDTH = '38px';
+const LEAD_BY_EXAMPLE_GRID_ROW_GAP = '29px';
+const LEAD_BY_EXAMPLE_GRID_COLUMN_GAP = '48px';
+const LEAD_BY_EXAMPLE_TITLE_BOTTOM_MARGIN = '40px';
+const LEAD_BY_EXAMPLE_TITLE_MOBILE_BOTTOM_MARGIN = '29px';
```

Comment:
Some of these aren't even numbers, is that from the Figma? Typically they are even numbers iirc

Reply:
spoke to Mike about this. will update the 29px to 24px!